src/index.js

```diff
@@ -16,7 +16,10 @@ const getBaladyiatsForWilaya = require('./api/getBaladyiatsForWilaya');
 const getBaladyiatsForDaira = require('./api/getBaladyiatsForDaira');
 const getWilayaByBaladyiaName = require('./api/getWilayaByBaladyiaName');
 const getDairaByBaladyiaName = require('./api/getDairaByBaladyiaName');
-
+const getDairaByCodeAndWilayaCode = require('./api/getDairaByCodeAndWilayaCode');
+const getBaladiyaByCodeAndDairaCodeAndWilayaCode = require('./api/getBaladiyaByCodeAndDairaCodeAndWilayaCode');
+const getBaldiyatsForDairaByCodeForWilayaByCode =  require('./api/getBaldiyatsForDairaByCodeForWilayaByCode');
+const getBaldiyaByCodeForWilaya = require('./api/getBaldiyaByCodeForWilaya');
 const data = require('../data/WilayaList.json');
```

Comment:
I'd leave a blank line before the dataset import just for the sake of clarity and readablity